.eslintrc

```diff
@@ -56,20 +59,24 @@
     {
       "files": ["**/*.test.{ts,tsx}"],
       "rules": {
+        "jest/no-truthy-falsy": "off",
```

Comment:
we use isTruthy() / isFalsy() A LOT and I don't want to go and refactor everything